### ISSUE TITLE
Cushions Don't Display Hwyla Info 

### ISSUE THREAD

**Issue by WenXin20** [ercanserteli/xercamods#2]
`Minecraft v1.14.4`
`Forge v28.1.96`
`Hwyla v1.10.6-B67`
`Xerca v1.2`.

If Hwyla is installed, no information will display for the cushion except for a tiny square at the top. 

![2019-11-24_19 31 06](https://user-images.githubusercontent.com/12000593/70176277-d3f5cf80-169d-11ea-9969-7691e8635e60.png)

**Comment by ercanserteli** [comment]
I checked this Hwyla mod and unfortunately this cannot be fixed from my side. The problem is that Hwyla only adds overlay info for living entities, hanging entities (like paintings) and minecarts. It doesn't display anything for any entity that does not fit these types, including vanilla boats. 

![resim](https://user-images.githubusercontent.com/9050561/71308080-e0518a80-2408-11ea-8943-c6ddf2efb798.png)

I added a name entry for Cushion in the lang file in the new XercaMod version, so now if they fix this by displaying generic entities' names, it will also work with the Cushion.
